dockers/docker-platform-monitor/docker_init.j2

```diff
@@ -116,7 +117,20 @@ if [ -e $FANCONTROL_CONF_FILE ]; then
 fi
 
 if [ -e $MODULAR_CHASSISDB_CONF_FILE ]; then
-    IS_MODULAR_CHASSIS=1
+    if [ -e $PLATFORM_ENV_CONF_FILE ] && [ -e "/etc/sonic/chassisdb.conf" ]; then
```

Comment:
please add comments to explain the file checks here as well.

Reply:
> please add comments to explain the file checks here as well.

Add comments.